README.md

```diff
@@ -1,11 +1,11 @@
 ## eslint-config-digitalbazaar
 
-This package provides Digital Bazaar's .eslintrc.js as an extensible shared config.
-
-
-## Usage
-We only export one config that can be used to extend your root .eslintrc.js
+This package provides Digital Bazaar's eslint rules as a set of extendable shared configs.
 
+We have 3 rule sets currently:
```

Comment:
Replace use of we/our throughout.

Reply:
this one is a little subjective so please do review the read me and mark as resolved if you are happy with the updated copy.